src/home/home.scss

```diff
@@ -119,6 +119,10 @@
         text-align: center;
         padding: 2em;
       }
+      & .heading3 {
```

Comment:
you can override heading1 (from styles.scss) here.  Note it is possible that the lineheight is actually what you want to change here, not the margin.  Check in the mock.